iOS CI Build fail on pull request

**Idea Description**

> SessionView.swift:4:8: error: no such module 'Model'

https://github.com/DroidKaigi/conference-app-2022/actions/runs/3077179403/jobs/4971999315

**Reference images and links**

--- a/app-ios/Package.swift
+++ b/app-ios/Package.swift
@@ -151,6 +151,7 @@ var package = Package(
             dependencies: [
                 .target(name: "appioscombined"),
                 .target(name: "Assets"),
+                .target(name: "Model"),
                 .target(name: "Theme"),
                 .product(name: "ComposableArchitecture", package: "swift-composable-architecture"),
             ]

